src/driver/types/ColumnTypes.ts

```diff
@@ -208,6 +213,7 @@ export type SimpleColumnType =
     | "array" // cockroachdb, sap, spanner
     | "cube" // postgres
     | "ltree" // postgres
+    | "vector" // postgres
```

Comment:
Not required in `SimpleColumnType` if you have already added it to `VectorColumnType` / `WithLengthColumnType`.

Also: can we add both `vector` and `halfvec`?